### PR TITLE
Added support to get conversation list mode and messages from a numbe…

### PR DESCRIPTION
**Pull request by rsanjuan87**
Added support to get conversation list mode and messages from a number, 
'-f <number>' param to get from specefic phone number, 
'-c' param to get a conversation list mode

depends on
https://github.com/termux/termux-api/pull/407

### CODE REVIEW

File: scripts/termux-sms-list [hunk 1]
```diff
@@ -3,7 +3,8 @@ set -u
 
 PARAM_LIMIT=10
 PARAM_OFFSET=0
-PARAM_TYPE=inbox
+PARAM_TYPE= all
```
Comment (Grimler91) [inline scripts/termux-sms-list:6]: extra space here

File: scripts/termux-sms-list [hunk 2]
```diff
@@ -30,6 +34,9 @@ do
         n) PARAMS="$PARAMS --ez show-phone-numbers true";;
         o) PARAM_OFFSET=$OPTARG;;
         t) PARAM_TYPE=$OPTARG;;
+
+        u) PARAMS="$PARAMS --ez conversation-list true";;
+        f) PARAM_NUMBER=" --es from $OPTARG";;
```
Comment (Grimler91) [inline scripts/termux-sms-list:39]: New options need to be added to `while getopts :hdl:t:no: option` as well

File: scripts/termux-sms-list [hunk 3]
```diff
@@ -30,6 +34,9 @@ do
         n) PARAMS="$PARAMS --ez show-phone-numbers true";;
         o) PARAM_OFFSET=$OPTARG;;
         t) PARAM_TYPE=$OPTARG;;
+
+        u) PARAMS="$PARAMS --ez conversation-list true";;
```
Comment (Grimler91) [inline scripts/termux-sms-list:38]: Should be `c` here according to help message?

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Hi, thanks for the PR!  Some small things need to be changed here, see comments.

**Comment by rsanjuan87** [comment]
done
@Grimler91

**Comment by rsanjuan87** [comment]
remember it depends on https://github.com/termux/termux-api/pull/407

**Pull request merged**